apps/website-25/src/pages/api/courses/exercises/[exerciseId]/response.ts

```diff
@@ -0,0 +1,88 @@
+import { z } from 'zod';
+import createHttpError from 'http-errors';
+import db from '../../../../../lib/api/db';
+import { makeApiRoute } from '../../../../../lib/api/makeApiRoute';
+import {
+  ExerciseResponse,
+  exerciseResponseTable,
+} from '../../../../../lib/api/db/tables';
+
+export type GetExerciseResponseResponse = {
+  type: 'success',
+  exerciseResponse: ExerciseResponse,
+};
+
+export type PutExerciseResponseRequest = {
+  response: string,
+  completed?: boolean,
+};
+
+export default makeApiRoute({
+  requireAuth: true,
+  requestBody: z.optional(
+    z.object({
+      response: z.string(),
+      completed: z.boolean().optional(),
+    }),
+  ),
+  responseBody: z.object({
+    type: z.literal('success'),
+    exerciseResponse: z.any().optional(),
```

Comment:
```suggestion
    exerciseResponse: z.any(),
```

I don't see a codepath where this doesn't get returned on a successful response?